services/docsrs/docsrs.tester.js

```diff
@@ -14,5 +14,5 @@ t.create('Getting latest version works')
   .get('/rand/latest.json')
   .expectBadge({
     label: 'docs',
-    messsage: Joi.allow('passing', 'failing'),
+    message: Joi.allow('passing', 'failing'),
```

Comment:
:smile:

Reply:
Ha. Spelling common words is surprisingly difficult sometimes

Reply:
tbh I was slightly surprised this only kicked up the one